metriq_gym/schemas/examples/bseq.example.json

```diff
@@ -1,5 +1,5 @@
 {
     "benchmark_name": "BSEQ",
-    "shots": 10
+    "shots": 1000
```

Comment:
Probably a leftover from testing – let's keep the example with low number of shots; we can add "production" values in the suite definitions